lib/ansible/plugins/terminal/enos.py

```diff
@@ -1,34 +1,23 @@
-# This code is part of Ansible, but is an independent component.
-# This particular file snippet, and this file snippet only, is BSD licensed.
-# Modules you write using this snippet, which is embedded dynamically by
-# Ansible still belong to the author of the module, and may assign their own
-# license to the complete work.
 #
-# Copyright (C) 2017 Lenovo, Inc.
+# Copyright (C) 2018 Lenovo, Inc.
```

Comment:
I don't think you want to change the copyright date, as this means that code written in the 2017 year is no longer under copyright.